### PR TITLE
Feature/popup infection

### PR DESCRIPTION
**Pull request by TheCrazyKing**
This adds a dialog box (or popup) that will notify the user about how many infected persons he last met. In case he met no one infected, it will display an encouraging message saying he met no one infected. 

My question for the reviewers : how do you find the dising of the popup ? For now on it is minimalist; it's not my strength to do something that looks nice ahah

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sdp/TestTools.java [hunk 1]
```diff
@@ -123,4 +125,18 @@ static void resetSickCounter(){
...
     }
+    static void clickBack(){
+        clickBack(1000);
+    }
+
+    /**
+     * Will click on the back button of the phone and wait before and after
+     * @param waitTime
+     */
+    static void clickBack(int waitTime){
```
Comment (TheBlueHawk) [inline app/src/androidTest/java/ch/epfl/sdp/TestTools.java:136]: Very useful testing tool 👍

**Review by TheBlueHawk — APPROVED** [review]
Good PR, well implemented and tested. The design looks good, if needed we can always improve it in a future PR, but for now is more than enough :)

File: app/src/androidTest/java/ch/epfl/sdp/TestTools.java (hunk 1 above)
Comment (TheCrazyKing) [reply app/src/androidTest/java/ch/epfl/sdp/TestTools.java:136]: haha in fact

File: app/src/androidTest/java/ch/epfl/sdp/contamination/ConcreteAnalysisTest.java [hunk 2]
```diff
@@ -363,6 +369,11 @@ public void infectionProbabilityIsUpdated() throws Throwable {
         for (int i = 0; i < 5; i++) {
             city.get(badLocation).put(nowMillis+i*1000, Collections.singleton(new Layman(UNKNOWN, .99f + i)));
         }
+        city.get(badLocation).put(nowMillis+13,Collections.singleton((new Layman(INFECTED))));
+        city.get(badLocation).put(nowMillis+14,Collections.singleton(man1));
+        city.get(badLocation).put(nowMillis+12,Collections.singleton((new Layman(INFECTED,"Joseph"))));
+        city.get(badLocation).put(nowMillis+11,Collections.singleton((new Layman(INFECTED,"Amélie Poulain"))));
```
Comment (lucieho) [inline app/src/androidTest/java/ch/epfl/sdp/contamination/ConcreteAnalysisTest.java:375]: We hope she recovers fast.

**Review by lucieho — APPROVED** [review]
Nice feature! The design is simple and clear. Could add some color/change font style if yo want and if time allows it, but it's already a good job.

File: app/src/androidTest/java/ch/epfl/sdp/contamination/ConcreteAnalysisTest.java (hunk 2 above)
Comment (TheCrazyKing) [reply app/src/androidTest/java/ch/epfl/sdp/contamination/ConcreteAnalysisTest.java:375]: yeah she's in good hands

**Pull request merged**